staging/com.smartnews/manifest.tt

```diff
@@ -0,0 +1,37 @@
+class @com.smartnews
+{
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none();
+
+  list query top_articles(in opt counter: Number #[prompt="How many news articles do you want?"],
```

Comment:
- This query should be called generically `article`, as it is the "main query" to retrieve the article database. It can still return the top articles by default (ie if queries without any filter)
- Presumably this query should be monitorable? I don't see a reason why, and the user will want to be notified of certain news articles.
- `#_[prompt]` needs to be a _translatable_ annotation with `#_[]` instead of `#[]`. Also, it should be all-lowercase and have no question mark.